src/typings-new-format/spec-1.0/datamodel.ts

```diff
@@ -122,20 +127,26 @@ export interface TypeDeclaration extends common.Annotable {
      * If displayName is not specified, it defaults to the element's key
      * (the name of the property itself
      */
-    displayName?: string
+    displayName: string
```

Comment:
It is inconsistent with other `displayName` declarations here. I can't find any other `displayName` declaration that is required property. Is there a reason for that?